lib/cmd/internal/df.js

```diff
@@ -18,7 +18,7 @@ df.usage = "\nfhc df ping"
 var log = require("../../utils/log");
 var fhc = require("../../fhc");
 var util = require('util');
-var request = require('request').defaults({'proxy': fhc.config.get("proxy")});
+var request = require('request');   // don't set proxy or other configured defaults here, since the config may not be loaded when this module is
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingComment)